app/assets/javascripts/pretty-text/engines/discourse-markdown/watched-words-links.js

```diff
@@ -0,0 +1,122 @@
+function isLinkOpen(str) {
+  return /^<a[>\s]/i.test(str);
+}
+
+function isLinkClose(str) {
```

Comment:
```suggestion
function isLinkClosed(str) {
```

Reply:
I kept the original name from linkify:

https://github.com/markdown-it/markdown-it/blob/master/lib/rules_core/linkify.js#L14